web/META-INF/resources/js/services.js

```diff
@@ -0,0 +1,101 @@
+'use strict';
+
+/* Services */
+
+var mudrodServices = angular.module('mudrodServices', ['ngResource']);
```

Comment:
@gxtchen I see that none of the endpoints from [OntologyResource](https://github.com/mudrod/mudrod/blob/master/service/src/main/java/gov/nasa/jpl/mudrod/services/ontology/OntologyResource.java) are registered.
Is there any reason for this? I **think** I saw from some notes that you may have been having issues with the Ontology resources... is this the case?